### DISCUSSION TITLE
Scaling up to China

### DISCUSSION THREAD

**Discussion by maiieul** [planetscale/discussion#272] [FAQs] [answered]
Hello, I'm building a product that I think has potential in the Chinese market, but I saw that [PlanetScale is not available inside of the People's Republic of China](https://github.com/planetscale/discussion/discussions/250#discussioncomment-3401077).

Does the Enterprise plan offer support for operating in China? Are there any plans on opening a Chinese region? I found [this article the blog about the managed cloud service for Enterprises](https://planetscale.com/blog/introducing-planetscale-managed-cloud). Can you confirm it would work for China?

**Comment by fooforge** [comment, marked as answer]
Hey @maieulChevalier, Thank you for your interest in PlanetScale!

Would you mind reaching out via https://planetscale.com/contact? We're happy to have a conversation with you about our Managed Cloud offering through that channel.

**Reply by u007** [reply]
do you guys provide it?

**Reply by shvamabps** [reply]
Did this. Haven't received a reply yet.

**Comment by bytrix** [comment]
Hi Maieul, have you found the solution?

**Comment by samlambert** [comment]
Hi. There are currently no plans for PlanetScale to operate in china.
